respec/Inleiding.md

```diff
@@ -25,6 +40,7 @@ Het profiel dient tenminste aan de volgende richtlijnen te voldoen:
 - Laagdrempelig door modulair opbouw en duidelijk in vertalingen en uitleg.
```

Comment:
```suggestion
- Laagdrempelig door modulaire opbouw en duidelijkheid in vertalingen en uitleg.
```

Reply:
Maar welke modules hebben we het eigenlijk over?
Bij TOOI onderscheiden we vier soorten modules:
- ontologie
- thesauri
- waardelijsten
- registers

Of bedoelen we met modules de vier use cases uit sectie 2.2? Of de bouwstenen? Daar staat: 
> Het profiel is modulair opgebouwd. Hieronder beschrijven we 4 uses cases die overeenkomen met verschillende niveaus van diepgang waarop concepten in samenhang kunnen worden beschreven. Iedere use case wordt ondersteund door een bouwsteen in het profiel.